### PR TITLE
Refactor fs subcommands into fs command

### PR DESCRIPTION
**Pull request by grubernaut**
Refactors `nomad fs` command to eliminate `fs` subcommands.
Automatically displays the file, if the path specified is a file; lists
the directory if the path is a directory; or displays stat information
if the `-stat` flag is set.

Currently running `nomad fs ls` to find a file, then running the exact
same command with the exception of `cat` instead of `ls` is time
consuming and awkward. This allows operational testing to be greatly
enhanced, and makes our lives so much better.

### CODE REVIEW

**Comment by diptanu** [comment]
@grubernaut Can you also please update the docs for this change?

**Comment by grubernaut** [comment]
@diptanu done!

File: command/fs.go [hunk 1]
```diff
@@ -2,27 +2,222 @@ package command
...
 )
 
 type FSCommand struct {
 	Meta
 }
 
 func (f *FSCommand) Help() string {
-	return "This command is accessed by using one of the subcommands below."
+	helpText := `
+Usage: nomad fs <alloc-id> <path>
```
Comment (dadgar) [inline command/fs.go]: Fix indents to be in-line with the rest of the commands

File: command/fs.go [hunk 2]
```diff
@@ -2,27 +2,222 @@ package command
...
+	flags.BoolVar(&job, "job", false, "")
+	flags.BoolVar(&stat, "stat", false, "")
+
+	if err := flags.Parse(args); err != nil {
+		return 1
+	}
+	args = flags.Args()
+
+	if len(args) < 1 {
+		f.Ui.Error("allocation id is a required parameter")
```
Comment (dadgar) [inline command/fs.go]: This message isn't right if they set the job flag

File: website/source/docs/commands/fs.html.md.erb [hunk 3]
```diff
@@ -8,45 +8,44 @@ description: >
...
-client. The following subcommands are available - `cat`, `ls` and `stat`
+The `fs` command allows a user to navigate an allocation directory on a Nomad
+client. The following functionalities are available - `cat`, `ls` and `stat`
 
-`cat`: Reads contents of files and writes them to the standard output.
-`ls`: Displays the name of a file and directories and their associated information.
-`stat`: Displays information about a file.
+`cat`: If the target path is a file, Nomad will cat the target path.
+`ls`: If the target path is a directory, Nomad displays the name of a file and directories and their associated information.
+`stat`: If the `-s` flag is used, Nomad will Display information about a file.
```
Comment (dadgar) [inline website/source/docs/commands/fs.html.md.erb]: `-s` isn't the flag. It is defined to `-stat`

**Comment by dadgar** [comment]
LGTM

File: command/fs.go (hunk 2 above)
Comment (grubernaut) [inline command/fs.go]:
```
-> % nomad fs -job example
Mode        Size  Modfied Time           Name
drwxrwxr-x  22 B  16/05/16 13:10:58 EDT  alloc/
drwxrwxr-x  16 B  16/05/16 13:10:58 EDT  redis/
```

**Comment by grubernaut** [comment]
@dadgar PTAL

**Pull request merged**